nominal/nominal.py

```diff
@@ -92,42 +99,107 @@ def upload_tdms(
     import pandas as pd
     from nptdms import TdmsChannel, TdmsFile, TdmsGroup
 
-    path = Path(file)
-    with TdmsFile.open(path) as tdms_file:
-        # identify channels to extract
-        channels_to_export: dict[str, TdmsChannel] = {}
-        group: TdmsGroup
-        for group in tdms_file.groups():
-            channel: TdmsChannel
-            for channel in group.channels():
-                # some channels may not have the required properties to construct a time track
-                if ("wf_increment" in channel.properties) and ("wf_start_time" in channel.properties):
+    def _tdms_upload_with_time_column():
+        path = Path(file)
+        with TdmsFile.open(path) as tdms_file:
+            channels_to_export: dict[str, TdmsChannel] = {}
+            group: TdmsGroup
+            df = None
+
+            for group in tdms_file.groups():
+                time_channel = None
+                channel: TdmsChannel
+
+                # pull out timestamp column if expected
+                for channel in group.channels():
+                    if channel.name == timestamp_column:
+                        time_channel = channel
+                if not time_channel:
+                    logger.info(
+                        "Skipping channel group \"%s\" because expected timestamp_column \"%s\" does not exist",
+                        channel.group_name, timestamp_column)
```

Comment:
I'm not opposed to using the logger interpolation but everywhere else in our python code we're using f-strings, so might as well keep that convention

Reply:
@alkasm I pulled this logger syntax from elsewhere in the Python client lib